--- dcron-4.6.0.orig/defs.h
+++ dcron-4.6.0/defs.h
@@ -52,7 +52,7 @@
 #define SCRONTABS	"/etc/cron.d"
 #endif
 #ifndef PIDFILE
-#define PIDFILE		"/var/run/crond.pid"
+#define PIDFILE		"/run/crond.pid"
 #endif
 #ifndef CRONTABS
 #define CRONTABS	"/var/spool/cron/crontabs"
@@ -87,7 +87,7 @@
 #define SENDMAIL_ARGS	"-t", "-oem", "-i"
 #endif
 #ifndef PATH_VI
-#define PATH_VI		"/usr/bin/vi"	/* location of vi	*/
+#define PATH_VI		"/usr/bin/editor"	/* location of vi	*/
 #endif
 
 #ifndef ID_TAG
